Cargo.toml

```diff
@@ -40,6 +40,7 @@ semicolon_if_nothing_returned = "warn"
 type_complexity = "allow"
 undocumented_unsafe_blocks = "warn"
 unwrap_or_default = "warn"
+allow_attributes_without_reason = "warn"
```

Comment:
This needs to be removed for now to get the PR mergeable.

Reply:
Ah, got it. Will do.